### PR TITLE
Throw error via graphql when tip is stale

### PR DESCRIPTION
**Pull request by limebell**
Additionally, it became not to use `RPCException` in graphql. Now uses `NodeExceptionType` instead.

### CODE REVIEW

**Review by riemannulus — APPROVED** [review]

File: Libplanet.Standalone/Hosting/LibplanetNodeService.cs [hunk 1]
```diff
@@ -420,9 +449,11 @@ protected async Task CheckPeerTable(CancellationToken cancellationToken = defaul
                 {
                     if (grace == count)
                     {
-                        var message = "No any peers are connected even seed peers were given.";
-                        _exceptionHandlerAction(RPCException.NetworkException, message);
-                        Properties.NodeExceptionOccurred((int)RPCException.NetworkException, message);
+                        var message = "No any peers are connected even seed peers were given. " +
+                                     $"(grace: {grace}";
+                        Log.Error(message);
+                        // _exceptionHandlerAction(RPCException.NetworkException, message);
```
Comment (longfin) [inline Libplanet.Standalone/Hosting/LibplanetNodeService.cs:455]: Is it intended?

File: NineChronicles.Standalone/GraphTypes/NodeExceptionType.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿using GraphQL.Types;
+using Libplanet.Standalone;
```
Comment (longfin) [inline NineChronicles.Standalone/GraphTypes/NodeExceptionType.cs:2]: Is this using needed?

File: NineChronicles.Standalone/GraphTypes/StandaloneSubscription.cs [hunk 3]
```diff
@@ -13,6 +13,7 @@
 using Libplanet;
 using Libplanet.Blockchain;
 using Libplanet.Net;
+using Libplanet.Standalone;
```
Comment (longfin) [inline NineChronicles.Standalone/GraphTypes/StandaloneSubscription.cs:16]: Is this using needed?

File: NineChronicles.Standalone/StandaloneContext.cs [hunk 4]
```diff
@@ -2,6 +2,7 @@
 using Libplanet.Blockchain;
 using Libplanet.KeyStore;
 using Libplanet.Net;
+using Libplanet.Standalone;
```
Comment (longfin) [inline NineChronicles.Standalone/StandaloneContext.cs:5]: Is this using needed?

**Review by longfin — APPROVED** [review]

File: NineChronicles.Standalone/GraphTypes/NodeExceptionType.cs (hunk 2 above)
Comment (limebell) [reply NineChronicles.Standalone/GraphTypes/NodeExceptionType.cs:2]: It is needed because `NodeException` now in `Libplanet.Standalone`.

**Pull request merged**